Shared/Samples/Play KML tour/PlayKMLTourView.swift

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct PlayKMLTourView: View {
+    /// A scene with an imagery basemap and a world elevation service.
+    @State private var scene: ArcGIS.Scene = {
+        let scene = Scene(basemapStyle: .arcGISImagery)
+        scene.initialViewpoint = Viewpoint(latitude: .nan, longitude: .nan, scale: 114_145_911)
+        
+        // Set the scene's base surface with an elevation source.
+        let surface = Surface()
+        let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+        surface.addElevationSource(elevationSource)
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// The current viewpoint of the scene view used when reseting the tour.
+    @State private var viewpoint: Viewpoint?
+    
+    /// The tour controller used to control the KML tour.
+    @State private var tourController = KMLTourController()
+    
+    /// The current status of the KML tour.
+    @State private var tourStatus: KMLTour.Status = .notInitialized
+    
+    /// A Boolean that indicates whether to show an error alert.
+    @State private var isShowingErrorAlert = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error? {
+        didSet { isShowingErrorAlert = error != nil }
+    }
+    
+    /// A Boolean value that indicates whether the KML tour is not ready to be played.
+    var tourIsNotReady: Bool {
```

Comment:
```suggestion
    private var tourIsNotReady: Bool {
```

Consider renaming the boolean to `tourDisabled`.